src/07-even-more-loops.js

```diff
@@ -57,7 +79,15 @@ function shoutForOfLoop(array) {}
  *  sumArray([ 10, 0, 10, 11 ]);
  *  //> 31
  */
-function sumArray(array) {}
+function sumArray(array) {
+  //let total = 0;
+  let arr = 0;
```

Comment:
This is a misleading variable name since the value the variable holds is a number, not an array. Could you pick a name that better represents what the variable is used for?